recipe/patches/fix-tests.patch

```diff
@@ -0,0 +1,65 @@
+diff -ruN netcdf-c-4.8.1/CMakeLists.txt netcdf-c-4.8.1-patch/CMakeLists.txt
+--- netcdf-c-4.8.1/CMakeLists.txt	2021-08-18 19:49:05.000000000 +0200
++++ netcdf-c-4.8.1-patch/CMakeLists.txt	2022-10-30 11:32:52.893519202 +0100
+@@ -941,17 +941,26 @@
+ # Check for the math library so it can be explicitly linked.
+ IF(NOT WIN32)
+   FIND_LIBRARY(HAVE_LIBM NAMES math m libm)
+-  MESSAGE(STATUS "Found Math library: ${HAVE_LIBM}")
+   IF(NOT HAVE_LIBM)
+-    MESSAGE(FATAL_ERROR "Unable to find the math library.")
++    CHECK_FUNCTION_EXISTS(exp HAVE_LIBM_FUNC)
++    IF(NOT HAVE_LIBM_FUNC)
++        MESSAGE(FATAL_ERROR "Unable to find the math library.")
++    ELSE(NOT HAVE_LIBM_FUNC)
++        SET(HAVE_LIBM "")
++    ENDIF()
++  ELSE(NOT HAVE_LIBM)
++    MESSAGE(STATUS "Found Math library: ${HAVE_LIBM}")
+   ENDIF()
+ ENDIF()
+ 
+ # Option to Enable DAP long tests, remote tests.
+ OPTION(ENABLE_DAP_LONG_TESTS "Enable DAP long tests." OFF)
++OPTION(ENABLE_EXTERNAL_SERVER_TESTS "Enable external Server remote tests." OFF)
+ OPTION(ENABLE_DAP_REMOTE_TESTS "Enable DAP remote tests." ON)
+ SET(REMOTETESTSERVERS "remotetest.unidata.ucar.edu" CACHE STRING "test servers to use for remote test")
+ 
++SET(REMOTETESTSERVERS "remotetest.unidata.ucar.edu" CACHE STRING "test servers to use for remote test")
++
+ # See if we have libzip
+ FIND_PACKAGE(Zip)
+ 
+diff -ruN netcdf-c-4.8.1/nc_test/CMakeLists.txt netcdf-c-4.8.1-patch/nc_test/CMakeLists.txt
+--- netcdf-c-4.8.1/nc_test/CMakeLists.txt	2021-08-18 19:49:05.000000000 +0200
++++ netcdf-c-4.8.1-patch/nc_test/CMakeLists.txt	2022-10-30 12:00:09.828975244 +0100
+@@ -94,11 +94,6 @@
+       add_sh_test(nc_test run_diskless2)
+     ENDIF()
+ 
+-    IF(ENABLE_BYTERANGE)
+-        build_bin_test_no_prefix(tst_byterange)
+-        add_sh_test(nc_test test_byterange)
+-    ENDIF()
+-
```

Comment:
Disabling test `nc_test_test_byterange` that is failing because of an inaccessible server.